receiver/filestatsreceiver/config.go

```diff
@@ -15,22 +15,28 @@
 package filestatsreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filestatsreceiver"
 
 import (
+	// scaffold-off
 	"errors"
 
 	"go.opentelemetry.io/collector/receiver/scraperhelper"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filestatsreceiver/internal/metadata"
+	// scaffold-on
 )
 
 type Config struct {
+	// scaffold-off
 	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
 	metadata.MetricsBuilderConfig           `mapstructure:",squash"`
 	Include                                 string `mapstructure:"include"`
+	// scaffold-on
```

Comment:
I'm afraid this partially generated can be hard to maintain. Can we first try to move some bits of existing copy-pasted code to `_generated.go` files?

Reply:
Sure, let me see what I can do.